packages/koehnlab/io/basis_util.py

```diff
@@ -0,0 +1,135 @@
+import numpy as np
+from spin_hamiltonians import spin_utils
+
+def get_propmat_prod(l,multiplicity:int,spat_num:int):
+        """
+        Returns the given l matrix in productbasis (spin-spatial basis).
+        Args:
+        ---------------------
+        l -- matrix of angular momentum in (kg*m^2)/s
+        multiplicity -- multiplicity of the system
+        spat_num - number of spatial states
+        Returns:
+        ---------------------
+        Lmat -- Matrix of angular momentum in productbasis in (kg*m^2)/s
```

Comment:
```suggestion
        l -- matrix of angular momentum in
        multiplicity -- multiplicity of the system
        spat_num - number of spatial states
        Returns:
        ---------------------
        Lmat -- Matrix of angular momentum in productbasis
```
Units are (usually) atomic units